### PR TITLE
remove reference to Assetic

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
At the end of the document we have this:

```rst
.. toctree::
    :maxdepth: 1
    :glob:

    frontend/*
```

This will include the `frontend/assetic.rst` file too. We should remove that too. Is it possible to exclude a file from the toctree? Otherwise, we will need to list all files to be actually included explicitly.

**Comment by javiereguiluz** [comment]
Try this, but I haven't tested it myself:

```rst
.. toctree::
    :hidden:

    frontend/assetic.rst

.. toctree::
    :maxdepth: 1
    :glob:

    frontend/*
```

**Comment by javiereguiluz** [comment]
Thanks Christian ... and thanks Assetic. We no longer need you ... but you were great!

File: frontend.rst [hunk 1]
```diff
@@ -93,6 +86,7 @@ Other Front-End Articles
     :hidden:
     :glob:
 
+    frontend/assetic
```
Comment (xabbuh) [inline frontend.rst]: Looks like this does not help. I also forgot to remove one paragraph. Both issues should be fixed with #10059.

**Pull request merged**